main.go

```diff
@@ -81,6 +81,7 @@ func main() {
 	flag.StringVar(&configMapName, "config", "codeflare-operator-config",
 		"The name of the ConfigMap to load the operator configuration from. "+
 			"If it does not exist, the operator will create and initialise it.")
+	flag.Parse()
```

Comment:
Shouldn't that be called after all the flags have been bound, i.e., after `zapOptions.BindFlags(flag.CommandLine)`?

Reply:
Of course! good catch! thanks
Added a fix